args/espers.py

```diff
@@ -6,7 +15,13 @@ def parse(parser):
     from data.characters import Characters
     espers = parser.add_argument_group("Espers")
 
+    esper_start = espers.add_mutually_exclusive_group()
+    esper_start.add_argument("-stesp", "--starting-espers", default = [0, 0], type = int,
+                                nargs = 2, metavar = ("MIN", "MAX"), choices = range(MAX_STARTING_ESPERS + 1),
+                                help = "Party starts with %(metavar) random espers")
```

Comment:
Super minor, but this shows up in -h as `Party starts with ('MIN', 'MAX')andom espers`. 

Change `%(metavar)` to '%(metavar)s`